src/ShopBundle/objects/TShopBasket/TShopBasketCore.class.php

```diff
@@ -168,14 +168,14 @@ class TShopBasketCore implements IDataExtranetUserObserver, IPkgCmsSessionPostWa
     /**
      * use GetActiveShippingGroup to access this property.
      *
-     * @var TdbShopShippingGroup
+     * @var TdbShopShippingGroup|null
      */
     private $oActiveShippingGroup = null;
 
     /**
      * use GetActivePaymentMethod to access the property.
      *
-     * @var TShopPaymentMethod
+     * @var TShopPaymentMethod|null
```

Comment:
This is still shown as (type) error as the method is annotated with TdbShopPaymentMethode (so a subclass) below.